### ISSUE TITLE
Could not find `config.cfg` while loading i2b2 datsets. 

### ISSUE THREAD

**Issue by MANISH007700** [medspacy/medspacy#132]
Hello Team, 
I am trying to use `Section Dection` for my project, and I see that there's an option for custom section json and can be added in sectionizer. 
When I refer the `2-Adding Section.ipynb` file, there;s this i2b2 datasets downloaded and loaded. 

When I try the same thing `nlp = spacy.load("en_info_3700_i2b2_2012")` , I get this : 
Remind you : The pip install worked fine 

Versions : 
py - 3.8.10 
spacy - 3.1.6

```
/home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py:770: UserWarning: [W094] Model 'en_info_3700_i2b2_2012' (0.1.0) specifies an under-constrained spaCy version requirement: >=2.2.2. This can lead to compatibility problems with older versions, or as new spaCy versions are released, because the model may say it's compatible when it's not. Consider changing the "spacy_version" in your meta.json to a version range, with a lower and upper pin. For example: >=3.1.6,<3.2.0
  warnings.warn(warn_msg)
OSError                                   Traceback (most recent call last)
/home/ubuntu/manish/medspacy/medspacy/notebooks/section_detection/2-Adding-Sections.ipynb Cell 7' in <cell line: 1>()
----> [1](vscode-notebook-cell://ssh-remote%2Bcareshare-algo-sandbox/home/ubuntu/manish/medspacy/medspacy/notebooks/section_detection/2-Adding-Sections.ipynb#ch0000006vscode-remote?line=0) nlp = spacy.load("en_info_3700_i2b2_2012")

File ~/.local/lib/python3.8/site-packages/spacy/__init__.py:51, in load(name, vocab, disable, exclude, config)
     [30](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=29) def load(
     [31](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=30)     name: Union[str, Path],
     [32](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=31)     *,
   (...)
     [36](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=35)     config: Union[Dict[str, Any], Config] = util.SimpleFrozenDict(),
     [37](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=36) ) -> Language:
     [38](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=37)     """Load a spaCy model from an installed package or a local path.
     [39](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=38) 
     [40](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=39)     name (str): Package name or model path.
   (...)
     [49](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=48)     RETURNS (Language): The loaded nlp object.
     [50](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=49)     """
---> [51](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=50)     return util.load_model(
     [52](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=51)         name, vocab=vocab, disable=disable, exclude=exclude, config=config
     [53](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/__init__.py?line=52)     )

File ~/.local/lib/python3.8/site-packages/spacy/util.py:349, in load_model(name, vocab, disable, exclude, config)
    [347](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=346)     return get_lang_class(name.replace("blank:", ""))()
    [348](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=347) if is_package(name):  # installed as package
--> [349](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=348)     return load_model_from_package(name, **kwargs)  # type: ignore[arg-type]
    [350](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=349) if Path(name).exists():  # path to model data directory
    [351](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=350)     return load_model_from_path(Path(name), **kwargs)  # type: ignore[arg-type]

File ~/.local/lib/python3.8/site-packages/spacy/util.py:382, in load_model_from_package(name, vocab, disable, exclude, config)
    [367](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=366) """Load a model from an installed package.
    [368](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=367) 
    [369](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=368) name (str): The package name.
   (...)
    [379](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=378) RETURNS (Language): The loaded nlp object.
    [380](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=379) """
    [381](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=380) cls = importlib.import_module(name)
--> [382](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=381) return cls.load(vocab=vocab, disable=disable, exclude=exclude, config=config)

File ~/.local/lib/python3.8/site-packages/en_info_3700_i2b2_2012/__init__.py:12, in load(**overrides)
     [11](file:///home/ubuntu/.local/lib/python3.8/site-packages/en_info_3700_i2b2_2012/__init__.py?line=10) def load(**overrides):
---> [12](file:///home/ubuntu/.local/lib/python3.8/site-packages/en_info_3700_i2b2_2012/__init__.py?line=11)     return load_model_from_init_py(__file__, **overrides)

File ~/.local/lib/python3.8/site-packages/spacy/util.py:544, in load_model_from_init_py(init_file, vocab, disable, exclude, config)
    [542](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=541) if not model_path.exists():
    [543](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=542)     raise IOError(Errors.E052.format(path=data_path))
--> [544](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=543) return load_model_from_path(
    [545](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=544)     data_path,
    [546](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=545)     vocab=vocab,
    [547](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=546)     meta=meta,
    [548](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=547)     disable=disable,
    [549](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=548)     exclude=exclude,
    [550](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=549)     config=config,
    [551](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=550) )

File ~/.local/lib/python3.8/site-packages/spacy/util.py:416, in load_model_from_path(model_path, meta, vocab, disable, exclude, config)
    [414](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=413) config_path = model_path / "config.cfg"
    [415](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=414) overrides = dict_to_dot(config)
--> [416](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=415) config = load_config(config_path, overrides=overrides)
    [417](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=416) nlp = load_model_from_config(config, vocab=vocab, disable=disable, exclude=exclude)
    [418](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=417) return nlp.from_disk(model_path, exclude=exclude, overrides=overrides)

File ~/.local/lib/python3.8/site-packages/spacy/util.py:575, in load_config(path, overrides, interpolate)
    [573](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=572) else:
    [574](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=573)     if not config_path or not config_path.exists() or not config_path.is_file():
--> [575](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=574)         raise IOError(Errors.E053.format(path=config_path, name="config.cfg"))
    [576](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=575)     return config.from_disk(
    [577](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=576)         config_path, overrides=overrides, interpolate=interpolate
    [578](file:///home/ubuntu/.local/lib/python3.8/site-packages/spacy/util.py?line=577)     )

OSError: [E053] Could not read config.cfg from /home/ubuntu/.local/lib/python3.8/site-packages/en_info_3700_i2b2_2012/en_info_3700_i2b2_2012-0.1.0/config.cfg
```

I also tried downloading the tar gz file and then extracting it and passing into spacy.load(), but no luck :(

Any help ? 
Thanks

**Comment by abchapman93** [comment]
Hello! Currently, we don't have a version of that i2b2 model which is compatible with spaCy 3. I would like to do that soon but for now it will only work with spaCy 2. 

I'd also like at some point to compile a list of openly available pre-trained models. If you look at #131, perhaps the scispaCy model referenced there could fit your purposes. Or perhaps [med7](https://github.com/medspacy/medspacy/blob/master/notebooks/06-Using-Pretrained-Models.ipynb) would work.

**Comment by burgersmoke** [comment]
@MANISH007700 I think that there is a simple solution for your needs.  I see that your goal was to run the notebook `2-Adding Section.ipynb`.

I fixed this notebook in another issue (#116 ).  The short answer is that the language model that is loaded in your errors above was not really necessary for what was being demonstrated in the notebook. This fix has not yet been pulled into the master branch, but if you simply want to run this notebook as a demonstration, could you try getting this revision to see if it unblocks you?

https://github.com/medspacy/medspacy/blob/issue_116_fix_adding_sections_notebook/notebooks/section_detection/2-Adding-Sections.ipynb

**Comment by MANISH007700** [comment]
Yes @burgersmoke. Later that day I saw another branch named "issue_116_......" and used that to run the notebook. But in that we load the spacy's en model which doesn't have context of medical info.

**Comment by burgersmoke** [comment]
@MANISH007700 Did you try using med7 instead in your use case?  If so, was it helpful?

If so, this might be relevant to a conversation that @turbosheep  and I were having about this notebook since we were looking for the best way to demonstrate behavior.

**Comment by burgersmoke** [comment]
I would love to hear comments about this, but since the original issue (the i2b2 dataset) is now resolved with the updated notebook in medspacy 1.0.0, please open any new issues as enhancements or suggestions.
